### DISCUSSION TITLE
Rework/amélioration des pets

### DISCUSSION THREAD

**Discussion by darkfailleur** [Crownicles/Crownicles#2090] [Nouvelles idées]
Le but serait d'essayer de rendre les pets plus intéressant a essayer d'optimise ( essayer d'avoir des pet plus rare, vraiment essayer de les dressés) 

Essayer de faire des pets une vrai mécanique et pas juste un équipement qui boost de rare mini event et servent de décoration pour le profil

Ce qui serait éventuellement possible ( je ne dit pas facile a développé je ne m'y connais pas assez dans le domaine ) serait de pouvoir organiser des combat un peu a la pokémon entre pets avec un classement, et la possibilité de renforcer leur rareté non plus en l'obtenant 4* de base par exemple mais en trouvant quelqu'un avec le même pet que nous et de la même rareté, et chacun de nous a par exemple 10% de chance que son pet passe de 3 a 4* ( ça empêche de farm avec la même personne en boucle vu que les 2 ne sont pas forcer de passé a la rareté suivante en même temps ) et la rareté augmenterais les stats en combats 

Voili voilou, je me rend pas compte du travail que ça pourrait nécessité, et si y a pas la possibilité c'était qu'une proposition pour améliorer la mécanique des pets

**Comment by Nargacaura** [comment]
Ça me semble assez complexe de modifier la rareté d'un pet hors dressage entier, en plus d'être similaire à ce qui a été proposé dans la discussion #1680.

**Comment by Syandre** [comment]
J'hésitais à poster ceci, qui me trotte dans la tête depuis plusieurs mois déjà et je vois que le sujet est assez proche, donc j'en profite.


### **Poursuivre l'entrainement des familiers pour en augmenter le compteur d'étoiles :**

En observant les copains ou le canal de vente de familiers, parfois certaines personnes recherchent simplement leur animal préféré plutôt que de chasser la rareté du familier 5⭐. 
L'idée serait de rajouter la possibilité de poursuivre l'entrainement des familiers une fois leur dressage accompli, afin d'en augmenter leur compteur d'étoiles (ou rareté). Ainsi, les fans de petits chats ou de pingouins (entre autre) pourraient à terme obtenir artificiellement un familier équivalent à un 5⭐. 
 
 Intérêts / idées autour :

- Obtenir des familiers entrainés au delà du statut "Dressé".
- Donner à terme une chance d'accès aux mini-évents basés sur la rareté du familier aux fans de certains animaux non 5⭐.
- Dépenser encore plus de thunes dans les pets 😅. (argent / gemmes ?)
- Redonner un peu d'intérêt à tout ces pauvres animaux fielleux tout nuls que personne n'aime à cause de leur grade (enfin ça, ça reste à voir ce n'est pas garanti 🙄).
- Lier leur entrainement à divers mini-évents (pour le meilleur ou pour le pire) ?
- Pourrait se greffer à d'autres idées d'améliorations / attaques des pets observées à côté ? Il y a déjà plusieurs sujets.


Sinon, je n'ai pas l'expertise pour le développement, mais ça ne ressemble pas qu'à une petite amélioration. Ça risque effectivement de demander pas mal de boulot autour des pets et de ce qu'il est possible de faire ou non autour du reste.
